hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterRpcMultiDestination.java

```diff
@@ -475,7 +475,7 @@ public void testCallerContextWithMultiDestinations() throws IOException {
 
     String auditFlag = "src=" + dirPath.toString();
     String clientIpInfo = "clientIp:"
-        + InetAddress.getLocalHost().getHostAddress();
+        + InetAddress.getByName("localhost").getHostAddress();
```

Comment:
Is this multiplatform or will only work on Linux?

Reply:
I've checked on Windows, works just fine. In Java doc there is no a note that it's Linux only: https://docs.oracle.com/javase/8/docs/api/java/net/InetAddress.html#getByName-java.lang.String-

Main:
`System.out.println(System.getProperty("os.name"));`
`InetAddress localhost = InetAddress.getByName("localhost");`
`System.out.println(localhost.getHostAddress());`

Output:
`Windows 10`
`127.0.0.1`